torchx/runtime/container/Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM pytorch/pytorch:1.13.0-cuda11.6-cudnn8-runtime
 
 WORKDIR /app
 
+# upgrade pip to 22.x+ which has a faster dependency resolver
```

Comment:
Is there an existing image with upgraded pip? This could further reduce build time.

Reply:
pip is pretty small -- upgrading is pretty fast